packages/e2e-test-app/windows/NuGet.Config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="repositoryPath" value="windows/packages" />
```

Comment:
@AgneLukoseviciute  Now that you've moved it you need to change this back to just `packages`.

Reply:
Nevermind I just went ahead and did it.